readme.md

```diff
@@ -100,38 +100,6 @@ To run all tests:
 yarn test
 ```
 
-### Development Flow
```

Comment:
Would be nice to still have some information here to explain how things work (for example, some distillation of the notes on this PR).